### ISSUE TITLE
PubChem Fingerprint Failed

### ISSUE THREAD

**Issue by peiyaoli** [mims-harvard/TDC#197]
**Describe the bug**

I tried to convert smiles to PubChem Fingerprints but failed:

NameError: name 'canonicalize' is not defined

I have double-checked the code, the bug is quite obvious:

function `canonicalize` needs to be imported from 

**To Reproduce**

**Expected behavior**

**Screenshots**

**Environment:**
- OS:
- Python version:
- TDC version:
- Any other relevant information:

**Additional context**

**Comment by kexinhuang12345** [comment]
Hi！Can you send codes to reproduce the error?

**Comment by peiyaoli** [comment]
Hi, @kexinhuang12345 

Here is my code:

```python
from tdc.chem_utils import MolConvert

convert = MolConvert(src="SMILES", dst="PubChem")
convert("B12B3B4B1C234")
```
Here is the exception info:

---------------------------------------------------------------------------
NameError                                 Traceback (most recent call last)
Cell In[5], line 1
----> 1 convert("B12B3B4B1C234")

File [~/miniconda3/envs/QQ/lib/python3.9/site-packages/tdc/chem_utils/featurize/molconvert.py:1017](https://vscode-remote+ssh-002dremote-002bjungpu35.vscode-resource.vscode-cdn.net/home/peiyao/Projects/solubility-conformal-prediction/notebooks/~/miniconda3/envs/UQ/lib/python3.9/site-packages/tdc/chem_utils/featurize/molconvert.py:1017), in MolConvert.__call__(self, x)
   1015 if type(x) == str:
   1016     if self.func != smiles2morgan:
-> 1017         return self.func(x)
   1018     else:
   1019         return self.func(x, radius=self._radius, nBits=self._nbits)

File [~/miniconda3/envs/QQ/lib/python3.9/site-packages/tdc/chem_utils/featurize/molconvert.py:985](https://vscode-remote+ssh-002dremote-002bjungpu35.vscode-resource.vscode-cdn.net/home/peiyao/Projects/solubility-conformal-prediction/notebooks/~/miniconda3/envs/UQ/lib/python3.9/site-packages/tdc/chem_utils/featurize/molconvert.py:985), in MolConvert.__init__..(x)
    983     elif dst == "PubChem":
    984         f2 = smiles2pubchem
--> 985     self.func = lambda x: f2(f1(x))
    986 elif src in threeD_format:
    987     pass

File [~/miniconda3/envs/QQ/lib/python3.9/site-packages/tdc/chem_utils/featurize/_smiles2pubchem.py:940](https://vscode-remote+ssh-002dremote-002bjungpu35.vscode-resource.vscode-cdn.net/home/peiyao/Projects/solubility-conformal-prediction/notebooks/~/miniconda3/envs/UQ/lib/python3.9/site-packages/tdc/chem_utils/featurize/_smiles2pubchem.py:940), in smiles2pubchem(s)
    939 def smiles2pubchem(s):
--> 940     s = canonicalize(s)
    941     try:
    942         features = calcPubChemFingerAll(s)

NameError: name 'canonicalize' is not defined

**Comment by kexinhuang12345** [comment]
Hi! Just fixed in ![08b86d1](https://github.com/mims-harvard/TDC/commit/08b86d185ec283231395189c7a73cad763c514c1)
